az-hadoop-jobtype-plugin/src/main/java/azkaban/jobtype/HadoopJobUtils.java

```diff
@@ -90,6 +90,7 @@ public class HadoopJobUtils {
 
   public static final String YARN_KILL_VERSION = "yarn.kill.version";
   // values of the yarn kill version flag
+  public static final String YARN_KILL_DISABLED = "disabled";
```

Comment:
Where will this value be specified and would it apply to all the Azkaban jobs ?

Reply:
By the nature of flow container, rolling out a feature flag requires setting the flag in the `common.properties` in az-cluster-config, create and activate a new config image, so as it can be applied.